mobileapp/components/game/Game.tsx

```diff
@@ -68,10 +73,26 @@ export const Game = () => {
       return;
     }
     const tile = findTileByCoordinate(game.board, { x, y });
+
     const { gameState, board } = updateGame(game.board, game.gameState, tile);
 
-    setGame({ gameState, board });
+    const url = new URL(`/game/${gameId}`, API_ENDPOINT);
+    fetch(url, {
+      method: "post",
+      headers: {
+        Accept: "application/json",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        played: `${x}-${y}`,
+      }),
+    })
+      .then((r) => r.json())
+      .then((json) => {
+        console.log(json);
```

Comment:
No need to do something with the response, we just call the api to notify that the game has changed